packages/grid/package.json

```diff
@@ -0,0 +1,13 @@
+{
+ "name": "@starryui/grid",
+ "version": "1.0.0",
+ "types": "index.d.ts",
+ "description": "",
+ "main": "index.js",
+ "scripts": {
+  "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Remove this `test` script as it's not needed

Reply:
Done